semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java

```diff
@@ -483,14 +494,24 @@ public static int getHttpResponseStatusCode(Event event) {
 
     Map<String, AttributeValue> attributeValueMap = event.getAttributes().getAttributeMap();
     if (attributeValueMap.get(RawSpanConstants.getValue(HTTP_REQUEST_HEADER_PATH)) != null) {
-      return Optional.ofNullable(
-              attributeValueMap.get(RawSpanConstants.getValue(HTTP_REQUEST_HEADER_PATH)).getValue())
-          .map(path -> Utils.sanitizePath(path));
```

Comment:
we are not sanitizing.

Reply:
yes we are not. this is an older version